reactDashboard/src/components/DataCharts/DataChartsConsts.js

```diff
@@ -4,6 +4,9 @@ export const TOTAL_DEATHS = "totalDeaths";
 export const DAILY_CASES = "dailyCases";
 export const DAILY_DEATHS = "dailyDeaths";
 
+export const LAST_DAY = "lastDay";
```

Comment:
I'm undecided about creating and using constants here. The existing ones were there to match user selectable options, whereas the new ones are solely for programmatic use. An alternative would be a lower level version of `calculateDateRange` which took the same parameters as moment's `subtract()`, and then have the higher level version of the function call the lower level one, converting the passed in constant. The programmatic use would just call the lower level function directly/